README.md

```diff
@@ -88,7 +88,7 @@ Once TensorFlow's dependencies are installed, clone the `ngraph-bridge` repo:
 
         git clone https://github.com/tensorflow/ngraph-bridge.git
         cd ngraph-bridge
```

Comment:
@sayantan-nervana Can you also fix the typos @crlishka mentioned for the readme

Reply:
Fixed typos